packages/protobuf.rb

```diff
@@ -1,41 +1,47 @@
+# Adapted from Arch Linux protobuf PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/protobuf/trunk/PKGBUILD
+
 require 'package'
 
 class Protobuf < Package
-  description 'Protocol buffers are a language-neutral, platform-neutral extensible mechanism for serializing structured data.'
+  description 'Protocol Buffers - Googles data interchange format'
   homepage 'https://developers.google.com/protocol-buffers/'
-  version '3.9.2'
+  @_ver = '21.5'
```

Comment:
Wow, that is quite a jump.

Reply:
Right? They totally changed versioning...